spec/models/event_spec.rb

```diff
@@ -41,4 +41,30 @@
       it { is_expected.to_not be_valid }
     end
   end
+
+  describe '.requiring_notification' do
```

Comment:
Your spec should have a `subject` defined. the subject helper takes an optional symbol param allowing you to make more semantic expectations 
```ruby
subject(:smarties) { ... }

it 'sdfsdfsdf' do
  expect(:smarties).to be_better_than :m_and_ms
end
```